app/assets/javascripts/admin/addon/components/admin-config-areas/color-palette.gjs

```diff
@@ -63,6 +67,10 @@ export default class AdminConfigAreasColorPalette extends Component {
           message: i18n("saved"),
         },
       });
+      if (this.hasChangedColors) {
+        this.applyColorChangesIfPossible();
```

Comment:
```suggestion
        await this.applyColorChangesIfPossible();
```

Should we wait for this to complete before allowing another `submit`? Also, we might want to disable the submit button to prevent multiple `handleSubmit` actions from triggering at once. This is better UX and also allows us to wait until all changes have been applied before running our assertions in system tests.